service/models.py

```diff
@@ -67,13 +67,6 @@ def update(self):
             raise DataValidationError("Update called with empty ID field")
         db.session.commit()
 
-    def save(self):
```

Comment:
I removed this to bring code coverage up because it looks like the same method as def update(self)